core/bft/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Parity Technologies <admin@parity.io>"]
 
 [dependencies]
 futures = "0.1.17"
-parity-codec = { version = "2.0" }
+parity-codec = "~2.0.1"
```

Comment:
Why `~` and bugfix version? Can't we manage the exact version in the `Cargo.lock`?